lib/web/desktop.go

```diff
@@ -385,6 +385,9 @@ func (h *Handler) performSessionMFACeremony(
 			return mfa.PromptFunc(func(ctx context.Context, chal *proto.MFAAuthenticateChallenge) (*proto.MFAAuthenticateResponse, error) {
 				codec := tdpMFACodec{}
 
+				if chal.WebauthnChallenge == nil {
+					return nil, trace.AccessDenied("Desktop Access requires WebAuthn MFA, please register a WebAuthn device to connect")
+				}
```

Comment:
SSO MFA is supported as well
```suggestion
				if chal.WebauthnChallenge == nil && chal.SSOChallenge == nil {
					return nil, trace.AccessDenied("only WebAuthn and SSO MFA methods are supported on the web for desktop access, please register a supported mfa method to connect to this server")
				}
```

Reply:
Hm maybe it isn't supported, I'll update this in a separate PR.